src/D2L.Bmx/D2L.Bmx.csproj

```diff
@@ -5,20 +5,19 @@
     <PublishAot>true</PublishAot>
     <OutputType>Exe</OutputType>
     <TargetFramework>net7.0</TargetFramework>
-    <ImplicitUsings>enable</ImplicitUsings>
-    <Nullable>enable</Nullable>
-    <WarningsAsErrors>nullable</WarningsAsErrors>
+    <InvariantGlobalization>true</InvariantGlobalization>
+    <RootNamespace>D2L.Bmx</RootNamespace>
```

Comment:
some IDE/build tool features work more nicely with this specified when the assembly name is different.